packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -1083,6 +1086,13 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                     id,
                     value,
                 };
+
+                // The unreferenced property in a snapshot tree entry indicates whether it is unreferenced. If the
+                // property is not present, the tree entry is considered referenced. If the property is present and is
+                // true (which is the only value it can have), the tree entry is considered unreferenced.
+                if (unreferenced) {
```

Comment:
Feels like there is no need for it to be conditional statement (i.e. no need for IF) - IMHO more readable and usually more performant (branches are expensive for CPU :))

Reply:
I wanted referenced tree entries to not have the property at all. Without the `if` statement, the entries that are referenced will always have the `unreferenced` flag with the value undefined. With the `if` statement, referenced entries will not have the flag at all.

Reply:
undefined properties are not serialized in JSON (need to double check), so I believe it's not an issue. I.e. I believe it's only state in memory (as there is indeed difference between missing property and undefined property)
Either way is fine. :)

Reply:
I did not know that :-). I tested it and yeah, JSON does not serialize undefined properties.